moped-database/metadata/tables.yaml

```diff
@@ -1293,6 +1332,120 @@
           - phase_name
           - phase_key
         filter: {}
+- table:
+    name: moped_proj_component_tags
+    schema: public
+  object_relationships:
+    - name: moped_component_tag
+      using:
+        foreign_key_constraint_on: component_tag_id
+    - name: moped_proj_component
+      using:
+        foreign_key_constraint_on: project_component_id
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        columns:
+          - component_tag_id
+          - is_deleted
+          - project_component_id
+    - role: moped-editor
+      permission:
+        check: {}
+        columns:
+          - component_tag_id
+          - is_deleted
+          - project_component_id
+  select_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - is_deleted
+          - component_tag_id
+          - id
```

Comment:
I noticed this in the work types PR too, and I like that the permissions are set to read-only for the primary key. I don't think that I've noticed before that this isn't true of all the tables (ex. `moped_proj_components`).

Reply:
thanks for flagging that! I definitely haven't been consistent in looking out for it. i'll make a backlog issue to do a sweep of our other tables 🔍

Reply:
- https://github.com/cityofaustin/atd-data-tech/issues/12246